content/docs/rfcs/16/README.md

```diff
@@ -78,7 +78,7 @@ The following structured types are defined for use on the Debug API:
 
 | Field | Type | Inclusion | Description |
 | ----: | :---: | :---: |----------- |
-| `listenStr` | `String` | mandatory | Address that the node is listening for |
+| `listenAddresses` | `Array`[`String`] | mandatory | Addresses that the node is listening for |
```

Comment:
```suggestion
| `listenAddresses` | `Array`[`String`] | mandatory | Listening addresses of the node |
```

Not your wording, but the dangling preposition has been bothering me for a while, so I'm taking advantage of the opportunity 😄